contracts/Nexus.sol

```diff
@@ -285,9 +299,20 @@ contract Nexus is INexus, EIP712, BaseAccount, ExecutionHelper, ModuleManager, U
     }
 
     /// Upgrades the contract to a new implementation and calls a function on the new contract.
+    /// @notice Updates two slots 1. 1967 slot and 2. address() slot in case if it's upgraded earlier from Biconomy V2 account.
```

Comment:
...as Biconomy v2 Account (proxy) reads implementation from the slot that is defined by its address.

Reply:
fixed